lib/experimental/class-gutenberg-rest-comment-controller.php

```diff
@@ -110,10 +239,11 @@ public function create_item_permissions_check( $request ) {
 			);
 		}
 
-		if ( ! $this->check_post_type_supports_block_comments( $post->post_type ) ) {
+		// Note: This is only relevant change for the backport.
+		if ( ! comments_open( $post->ID ) && ! $is_block_comment ) {
```

Comment:
Override general `comments_open` check. I'm not sure if we want to support a similar condition at the post level.

I think it's better to handle it separately.

Reply:
If we support that this we need some this in the ui. I feel like block comments are different from normal comments, not sure we need this check.